WasatchNET/Spectrometer.cs

```diff
@@ -2209,7 +2327,7 @@ protected virtual double[] getSpectrumRaw()
                 {
                     if (!currentAcquisitionCancelled && errorOnTimeout)
                         logger.error($"failed when reading subspectrum from 0x{spectralReader.EpNum:x2} ({id})");
-                    Thread.Sleep(10);
+                    Thread.Sleep(100);

```

Comment:
My inclination is this should be configurable

Reply:
Done.